### PR TITLE
Fix upgrade

### PR DESCRIPTION
**Pull request by DimaStebaev**

### CODE REVIEW

File: migrations/tools/verification.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export async function verify(contractName: string, contractAddress: string, cons
                 break;
             } catch (e) {
                 if (e instanceof Error) {
-                    if (e.toString().includes("Contract source code already verified")) {
+                    if (e.toString().includes("Already Verified")) {
```
Comment (payvint) [inline migrations/tools/verification.ts:16]: Maybe let's combine instead of replace

File: migrations/tools/gnosis-safe.ts [hunk 2]
```diff
@@ -202,6 +199,9 @@ export async function createMultiSendTransaction(ethers: Ethers, safeAddress: st
         "origin": "Upgrade skale-manager"  // Give more information about the transaction, e.g. "My Custom Safe app"
     }
 
+    console.log(JSON.stringify(txToSend, null, 4));
+    process.exit(0);
```
Comment (payvint) [inline migrations/tools/gnosis-safe.ts]: Should it be here?

File: migrations/tools/verification.ts (hunk 1 above)
Comment (DimaStebaev) [reply migrations/tools/verification.ts:16]: There is the same message each time. I don't see a reason why to combine.

File: migrations/tools/gnosis-safe.ts (hunk 2 above)
Comment (DimaStebaev) [reply migrations/tools/gnosis-safe.ts]: No.

**Review by payvint — APPROVED** [review]

**Review by yavrsky — APPROVED** [review]

**Review by gannakulikova — APPROVED** [review]

**Pull request merged**